public/app/features/dashboard/dashgrid/DashboardEmpty.tsx

```diff
@@ -0,0 +1,221 @@
+            Add visualization
+          </Button>
+        </div>
+        <div className={cx(styles.centeredContent, styles.others)}>
+          <div className={cx(styles.containerBox, styles.centeredContent, styles.rowContainer)}>
+            <div className={cx(styles.headerSection, styles.headerSmall)}>Add a row</div>
+            <div className={cx(styles.bodySection, styles.bodySmall)}>
+              Group your visualizations into expandable sections.
+            </div>
+            <Button
+              icon="plus"
+              fill="outline"
+              aria-label="Add new row"
+              onClick={() => {
+                reportInteraction('Create new row');
+                onCreateNewRow();
+              }}
+              disabled={!canCreate}
+            >
+              Add row
+            </Button>
+          </div>
+          <div className={cx(styles.containerBox, styles.centeredContent, styles.libraryContainer)}>
+            <div className={cx(styles.headerSection, styles.headerSmall)}>Import panel</div>
+            <div className={cx(styles.bodySection, styles.bodySmall)}>
+              Import visualizations that are shared with other dashboards.
+            </div>
+            <Button
+              icon="plus"
+              fill="outline"
+              aria-label="Add new panel from panel library"
+              onClick={() => {
+                reportInteraction('Add a panel from the panel library');
+                onAddLibraryPanel();
+              }}
+              disabled={!canCreate}
+            >
+              Import library panel
+            </Button>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const getStyles = (theme: GrafanaTheme2) => {
+  return {
+    wrapper: css({
+      label: 'dashboard-empty-wrapper',
+      flexDirection: 'column',
+      maxWidth: '920px',
+      gap: theme.spacing.gridSize * 4,
+    }),
+    containerBox: css({
+      label: 'container-box',
+      flexDirection: 'column',
+      boxSizing: 'border-box',
+      border: '1px dashed rgba(110, 159, 255, 0.5)',
+    }),
+    centeredContent: css({
+      label: 'centered',
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+    }),
+    visualizationContainer: css({
+      label: 'visualization-container',
+      padding: theme.spacing.gridSize * 4,
+    }),
+    others: css({
+      label: 'others-wrapper',
+      alignItems: 'stretch',
+      flexDirection: 'row',
+      gap: theme.spacing.gridSize * 4,
+    }),
+    rowContainer: css({
+      label: 'row-container',
+      padding: theme.spacing.gridSize * 3,
+    }),
+    libraryContainer: css({
+      label: 'library-container',
+      padding: theme.spacing.gridSize * 3,
+    }),
+    visualizationContent: css({
+      gap: theme.spacing.gridSize * 2,
+    }),
+    headerSection: css({
+      label: 'header-section',
+      fontWeight: 600,
+    }),
+    headerBig: css({
+      fontSize: '32px',
```

Comment:
yes, but it's not useful in those cases imo - There are no 32 and 20 options, and I'd have to use `theme.typography.h5` for the smaller body texts for Add row and Add library panel, which semantically doesn't make sense to me

Reply:
mmm no, me neither. isn't there any other? take into account this approach lets you modify all the app if suddenly 32px is not the desired one

Reply:
Yes, you're right it will be better to allow the single source of truth to apply here as well. I changed the header sections to h1 and h2, and the body sections to h4 and h5. It's better also because the typography we use uses `rem`, so in the long run any general changes will affect the details very well.
@Ijin08 would you take a look at this:
![image](https://user-images.githubusercontent.com/13227501/227164414-361a51a1-fa6e-4792-a9c5-765f8a9ad042.png)
and the conversation here: Is it alright to continue with those dimensions?

The typography options we have are here: https://github.com/grafana/grafana/blob/redesign/empty-dashboard/packages/grafana-data/src/themes/createTypography.ts#L118

Reply:
![image](https://user-images.githubusercontent.com/13227501/227165091-17fc7a03-59ae-4ed4-afb4-f38162885c2a.png)

Reply:
Awesome!